themes/default/ProfileInfo.template.php

```diff
@@ -771,8 +771,10 @@ function template_profile_block_contact()
 					$cf_show = true;
 				}
 
+				// removed $field['name'], ': ', as it was annoying
+
 				echo '
-					<li>', $field['name'], ': ', $field['output_html'], '</li>';
+					<li>', $field['output_html'], '</li>';
```

Comment:
Intended. It was horrifically ugly.

I just made sure all of the icons have accessible text and title attributes instead. Otherwise, screen readers will often read it twice anyway, and sighted people will recognize most of the icons as-is.